include/task.h

```diff
@@ -90,8 +95,24 @@ extern u8 gIwramHeap[0x2204];
 
 u32 TasksInit(void);
 void TasksExec(void);
+
+#if ENABLE_TASK_LOGGING
+#include <stdio.h>
+
+struct Task *TaskCreate(TaskMain taskMain, u16 structSize, u16 priority, u16 flags,
+                        TaskDestructor taskDestructor, const char *name);
+
+#define TaskCreate(taskMain, structSize, priority, flags, taskDestructor)               \
```

Comment:
This feels hacky? Couldn't this just be added to the `TaskCreate` function?

Reply:
Oh I see the issue with the line and file has to be in the place where it's called, I get it. I'm surprised C understands if you have a function with the same name as a macro

Reply:
I don't think it'd match that way, but I can try.

Reply:
It's a relatively common way to use macros for stuff like this.
And in my eyes it's the only way to store the name without going to every single declaration and #ifdef-ing them for the matching and nonmatching version.
(Also it's only debug code, so it doesn't matter _too_ much, I think?)

Reply:
No worries!